backends-velox/src/main/scala/org/apache/gluten/expression/ExpressionTransformer.scala

```diff
@@ -56,7 +55,9 @@ case class VeloxGetStructFieldTransformer(
     child: ExpressionTransformer,
     ordinal: Int,
     original: GetStructField)
-  extends UnaryExpressionTransformer {
+  extends BinaryExpressionTransformer {
```

Comment:
Could you explain why we need this change?

Reply:
because we need to transform both `child` and `ordinal`, so the children size should be two.  Before this change, it doesn't need to transform `ordinal`.